microcosm_flask/tests/conventions/fixtures.py

```diff
@@ -81,6 +83,19 @@ def csv_column_order(self):
         return column_order
 
 
+def pubsub_schema(fields):
```

Comment:
Is there a reason you're not making this part of the public API? I understand if the intent is to reduce coupling of terms, but it feels like this is the main usage, and we'd be copy-pasting it around. Unless your intent is more that we redefine the convention on every project, which I'm not sure I agree with.

Reply:
It's because I don't think `microcosm-flask` should know/deal with pubsub or any other use case of this; all it's doing is exposing extra schemas. That's why I went with a fairly generic API.
I was going to define this one in `marquez`

Reply:
👍